[2012.01.06 22:26:00] (Coverage): Unexpected error
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextEntry(HashMap.java:793)
	at java.util.HashMap$EntryIterator.next(HashMap.java:834)
	at java.util.HashMap$EntryIterator.next(HashMap.java:832)
	at java.util.HashMap.putAllForCreate(HashMap.java:435)
	at java.util.HashMap.<init>(HashMap.java:225)
	at com.intellij.rt.coverage.data.ProjectData$ClassesMap.asMap(ProjectData.java:360)
	at com.intellij.rt.coverage.data.ProjectData.getClasses(ProjectData.java:196)
	at com.intellij.rt.coverage.instrumentation.SaveHook.save(SaveHook.java:55)
	at com.intellij.rt.coverage.instrumentation.SaveHook.run(SaveHook.java:41)
	at java.lang.Thread.run(Thread.java:662)
